FreeBSD/usr/ports/net/ceph-devel/distinfo

```diff
@@ -0,0 +1,35 @@
+TIMESTAMP = 1486914967
```

Comment:
@tchaikov 
I sort of agree.
Certainly after I have had this rather lengthy review process for the port submision on:
    https://reviews.freebsd.org/D9584
They are not make my life any easier, and having to keep it upstreamed into Ceph would be more "service" to others in ceph as to how to build and package it.
But I'm not closing it for a bit of time to come, since I'm not certain that the port will be accepted in it's current state. Main pain point is /etc/ceph.
Ports are not allowed to modify /etc and/or store data in /etc.

So I'll have to rework configs/authkey stuff to be able to work with /usr/local/etc/ceph.
And then also import that knowledge into all kinds of scripts.
And get that accepted here, which will take quite some time.

So bottom line: lets keep this submission for the time being. It is a mark in history.
And once FreeBSD ports accepts the port, I'll close this PR.